crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -37,20 +37,20 @@ struct Fee(u128);
 const ACCOUNTS_PREFIX: &str = "accounts";
 const TRANSFER_BASE_FEE_STORAGE_KEY: &str = "transferfee";
 
-fn storage_key(address: &str) -> String {
+fn storage_key(address: &Address) -> String {
```

Comment:
This fundamentally changes access to storage, segregating it by prefixes. Is this correct?

Reply:
After syncing with the team: this is not correct and not what we want. The storage access must not care about the prefix, only about the bytes of the address (and hence the first 20 bytes of an account's public key).